### PR TITLE
Created SQL tables

### PR DESCRIPTION
**Pull request by tomaspsotkka**
Task: Do SQL from GRD

I created tables for each entity except the streaks. I didn't even include streaks in users table so when adding streaks table or whatever, don't forget to add a streak parameter into a users table.

### CODE REVIEW

File: SQL/tables.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
+create table users(
+    id serial primary key,
+    username varchar(16) not null unique,
```
Comment (sebinko) [inline SQL/tables.sql]: dont lmit it, we never said a reason to limit it

File: SQL/tables.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    password varchar(16) not null,
```
Comment (sebinko) [inline SQL/tables.sql]: also the same, dont limit whis

File: SQL/tables.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    email varchar(30) not null unique,
```
Comment (sebinko) [inline SQL/tables.sql]: again no limit

File: SQL/tables.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    emotion varchar(255) not null,
+    created_at timestamp default current_timestamp,
+    updated_at timestamp default current_timestamp,
+    user_id int,
+    foreign key (user_id) references users(id) on delete cascade
+);
+
+create table reactions(
+    user_id int,
+    emotion_id int,
```
Comment (sebinko) [inline SQL/tables.sql]: rather emotion_checkin_id, so it is explicit

File: SQL/tables.sql [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    emoji varchar(255) not null,
```
Comment (sebinko) [inline SQL/tables.sql]: why is emoji 255 characters long?

File: SQL/tables.sql (hunk 1 above)
Comment (tomaspsotkka) [reply SQL/tables.sql]: aight

File: SQL/tables.sql (hunk 5 above)
Comment (tomaspsotkka) [reply SQL/tables.sql]: alan told us that it's a "default" number when we don't know how many chars will we use

File: SQL/tables.sql [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    foreign key (user_id) references users(id) on delete cascade
+);
+
+create table reactions(
+    user_id int,
+    emotion_checkin_id int,
+    emoji varchar not null,
+    created_at timestamp default current_timestamp,
+    updated_at timestamp default current_timestamp,
+    primary key (user_id, emotion_id),
```
Comment (sebinko) [inline SQL/tables.sql]: you forgot to update it here

File: SQL/tables.sql [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    foreign key (user_id) references users(id) on delete cascade,
+    foreign key (emotion_id) references emotion_checkins(id) on delete cascade
```
Comment (sebinko) [inline SQL/tables.sql]: also here, did you try running it?

Comment (tomaspsotkka) [reply SQL/tables.sql]: no

**Review by sebinko — APPROVED** [review]
good

**Pull request merged**